clientgui/NoticeListCtrl.cpp

```diff
@@ -264,6 +277,18 @@ void CNoticeListCtrl::OnWebViewError( wxWebViewEvent& event ) {
 
     event.Skip();
 }
+#else
+void CNoticeListCtrl::OnLinkClicked(wxHtmlLinkEvent &event)
+{
+	wxString url = event.GetLinkInfo().GetHref();
+	if (!url.StartsWith(wxT("http://")) && !url.StartsWith(wxT("https://"))) {
+		event.Skip();
+		return;
+	}
+	event.Skip(); // Tell element not to follow link
```

Comment:
Bug. Contrary to the comment, it makes wxHTMLWindow open the link inside Manager.